SofascoreAcademy/SofascoreAcademy/View/MatchView.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  MatchView.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 16.03.2024..
+//
+
+import Foundation
+import SofaAcademic
+import UIKit
+
+class MatchView: BaseView {
+    
+    var matchModel: Match
+    
+    let timeCellView: UIView = UIView()
+    let time: UILabel = UILabel()
+    let minutes: UILabel = UILabel()
+    
+    let divider: UIView = UIView()
+    
+    let team1: UIView = UIView()
+    let team1Logo: UIImageView = UIImageView()
+    let team1Name: UILabel = UILabel()
+    
+    let team2: UIView = UIView()
+    let team2Logo: UIImageView = UIImageView()
+    let team2Name: UILabel = UILabel()
+    
+    let team1Score: UILabel = UILabel()
+    let team2Score: UILabel = UILabel()
+    
+    init(match: Match) {
+        matchModel = match
+        super.init()
+    }
+    
+    override func addViews() {
+        timeCellView.addSubview(time)
+        timeCellView.addSubview(minutes)
+        
+        addSubview(timeCellView)
+        addSubview(divider)
+        
+        team1.addSubview(team1Logo)
+        team1.addSubview(team1Name)
+        addSubview(team1)
+        
+        team2.addSubview(team2Logo)
+        team2.addSubview(team2Name)
+        addSubview(team2)
+        
+        addSubview(team1Score)
+        addSubview(team2Score)
+    }
+
+    override func styleViews() {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "HH:mm"
+        time.text = dateFormatter.string(from: matchModel.startTime)
+        time.font = Fonts.regularCondensedFont.withSize(Fonts.smallSize)
+        time.textColor = Colors.onSurfaceLv2
+        time.textAlignment = .center
+        
+        divider.backgroundColor = Colors.onSurfaceLv4
+        
+        team1Logo.image = matchModel.team1.logo
+        team2Logo.image = matchModel.team2.logo
+        
+        minutes.font = Fonts.regularCondensedFont.withSize(Fonts.smallSize)
+        minutes.textColor = Colors.onSurfaceLv2
+        minutes.textAlignment = .center
+        
+        team1Name.text = matchModel.team1.name
+        team1Name.font = Fonts.regularFont.withSize(Fonts.regularSize)
+        team1Name.textColor = Colors.onSurfaceLv1
+        
+        team2Name.text = matchModel.team2.name
+        team2Name.font = Fonts.regularFont.withSize(Fonts.regularSize)
+        team2Name.textColor = Colors.onSurfaceLv1
+        
+        team1Score.font = Fonts.regularFont.withSize(Fonts.regularSize)
+        team1Score.textColor = Colors.onSurfaceLv1
+        team1Score.textAlignment = .right
+        
+        team2Score.font = Fonts.regularFont.withSize(Fonts.regularSize)
+        team2Score.textColor = Colors.onSurfaceLv1
+        team2Score.textAlignment = .right
```

Comment:
Kad ovo pogledaš, tu ima dosta linija. Možda nije loše imati custom view kao što je navedeno gore koji će imati sve te style stvari definirano unutar sebe, i može se reusati.